frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -156,4 +156,4 @@ type VirtualMachineTemplatesPageProps = {
   match: match<{ ns?: string }>;
 };
 
-export { VirtualMachineTemplatesPage };
+export { labelPlural, VirtualMachineTemplatesPage };
```

Comment:
this should probably go to some other file like `./constants.ts`

Reply:
:+1: nice